contentmgr/gc.go

```diff
@@ -152,6 +151,17 @@ func (cm *ContentManager) UnpinContent(ctx context.Context, contid uint) error {
 		return err
 	}
 
+	if pin.AggregatedIn > 0 {
+		if cm.ZonesConsolidating[pin.AggregatedIn] {
+			return fmt.Errorf("unable to unpin content while zone is consolidating (pin: %d, zone: %d)", pin.ID, pin.AggregatedIn)
+		}
+		if err := cm.DB.Model(util.Content{}).
+			Where("id = ?", pin.AggregatedIn).
```

Comment:
will there be a possible scenario that a "aggregated" content is older than 3 months? @en0ma

Reply:
yeah, but there is no issue, as their contents are pinned, just that no deal is made yet

Reply:
@neelvirdy thinking about it, when a pin is unpinned, we should only remove it in these conditions;

- if it's not in an aggregate - remove
- if it's in an aggregate, but the aggregate is not pinned(not active) yet - remove it
- if it's in an aggregate and the aggregate is already pinned(active) - ignore it